src/index.js

```diff
@@ -185,6 +185,12 @@ async function parseMMD (...args) {
  */
 async function renderMermaid (browser, definition, outputFormat, { viewport, backgroundColor = 'white', mermaidConfig = {}, myCSS, pdfFit } = {}) {
   const page = await browser.newPage()
+  page.on('console', async (msg) => {
+    const msgArgs = msg.args()
+    for (let i = 0; i < msgArgs.length; ++i) {
+      console.log(await msgArgs[i].jsonValue())
+    }
+  })
```

Comment:
I did a quick test and this seems to work better, but the Puppeteer docs are a bit vague, so I'm not sure if it is better: https://pptr.dev/api/puppeteer.consolemessage

What do you think?

```suggestion
  page.on('console', (msg) => {
    console.log(msg.text())
  })
```

Reply:
Yeah, it works even better. Thank you.